hindi/Bhagat_Singh/bhagat_singh.md

```diff
@@ -0,0 +1,13 @@
+Bhagat Singh aka Shaheed Bhagat Singh (28 September 1907 – 23 March 1931) was an [Indian](https://simple.wikipedia.org/wiki/India) socialist and a revolutionary. He is considered to be one of the most influential revolutionaries of the [Indian Independence Movement](https://simple.wikipedia.org/wiki/Indian_independence_movement).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.